### PR TITLE
Fix #693 - Change sign in to sign in/up

### LINKED ISSUE #693: Private Relay home page is ambigous on sign-in sign-up
# About this Issue
As mentioned by @birdsarah currently the page only has `sign in` which can confuse the user to expecting that they must have an account with Relay before they can access Relay.

### PR DESCRIPTION
**Pull request by codemist**
Changed sign in button to "sign in/up" for clarity. Referred to this suggestion https://github.com/mozilla/fx-private-relay/issues/693#issuecomment-700187715. 

Preview:
image

### CODE REVIEW

File: static/scss/partials/main.scss [hunk 1]
```diff
@@ -379,7 +379,7 @@ button:active {
   opacity: 0.9;
 }
 
-.sign-in {
+.sign-in{
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:382]: ⚡ this specific space? 
```suggestion
.sign-in {
```

File: static/scss/partials/main.scss [hunk 2]
```diff
@@ -397,6 +397,7 @@ button:active {
   box-shadow: 0 2px 8px 0 rgba(14, 13, 26, 0.12), 0 3px 1px -2px rgba(7, 48, 114, 0.12), 0 2px 4px 0 rgba(34, 0, 51, 0.04), 0 0 0 1px rgba(32, 18, 58, 0.04);
 }
 
+
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:400]:
```suggestion
```

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
This looks great. It seems like there were some edits that were made to the `main.scss` that should be dropped. Can you revise your commit to not edit this file?

**Comment by groovecoder** [comment]
Code LGTM but when I tried to spot-check locally I had some weird CSS behavior.

**Comment by codemist** [comment]
@groovecoder What was the behaviour?

**Comment by groovecoder** [comment]
The VPN banner has messed up styling and the "Sign up" element has a border? Looks like this in the branch and when I tried to rebase it on `main` branch too.

image

**Comment by codemist** [comment]
That's really odd, I can't seem to replicate that issue since it looks fine for me. @maxxcrawford Is there perhaps something I wrote in the commits that would have messed with the VPN banner?

**Comment by maxxcrawford** [comment]
@groovecoder Try the following steps and let me know if this is still an issue? 

1. Delete/empty your `/static/css` directory entirely.
2. Delete/empty your `/node_modules` directory entirely.
3. Reinstall dependencies `$ npm install`
4. Run `$ python manage.py runserver` to compile all the things.

To verify, open up [http://127.0.0.1:8000/static/css/app.css](http://127.0.0.1:8000/static/css/app.css) and scroll to the bottom of the file. It should have statements for `.vpn-promo-` selectors.

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
We need to adjust the breakpoint for the header. Nav items are cropped in between `550px` and `660px`. See screenshots below: 

At `551px` – the bento box 🍱 isn't visible at all, and "sign in" is cut off some: 

image

At `660px`, the right margin from the bento box menu 🍱  starts to crop. I would target the transition that happens at `550px` to activate here instead. 

image

FWIW: I'm using [Responsive Design Mode](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode) in the inspector to walk through the breakpoints pixel-by-pixel.

File: static/scss/partials/main.scss [hunk 3]
```diff
@@ -387,6 +387,11 @@ button:active {
   white-space: nowrap;
 }
 
+.sign-up {
+  border:none;
+  color:inherit;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:392]: nit: spacing
```suggestion
  border: none;
  color: inherit;
```

**Review by maxxcrawford — APPROVED** [review]
One nit, but approved!

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
My bad. I jumped the gun – I thought you had resolved the tablet breakpoint issue. Blocking again. 🙈

**Comment by codemist** [comment]
Preview:

https://user-images.githubusercontent.com/13066134/127208512-4b3349a4-f62d-4d5b-93b2-8dcfea1a1671.mov



I accidentally pushed 660px in that pre-existing media query when I should've kept it the same - sorry about that, it was part of my fix process. My latest commit reverted it back.

File: static/scss/partials/main.scss [hunk 4]
```diff
@@ -2101,6 +2106,16 @@ input.input-has-error {
   }
 }
 
+@media screen and (max-width: 660px) {
+  .hide-mobile-link {
+    display: none !important;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2111]: Are the `!important` tags necessary?

File: static/scss/partials/main.scss [hunk 5]
```diff
@@ -2101,6 +2106,16 @@ input.input-has-error {
...
+  }
+
+  .show-mobile-link {
+    display: flex !important;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:2115]: Are the `!important` tags necessary?

**Review by maxxcrawford — CHANGES_REQUESTED** [review]

File: static/scss/partials/main.scss (hunk 5 above)
Comment (codemist) [reply static/scss/partials/main.scss:2115]: The original `.hide-mobile` class had it so I just carried it over - should I remove it?

Comment (maxxcrawford) [reply static/scss/partials/main.scss]: Yeah. `!important` should be super super rare.

Comment (codemist) [reply static/scss/partials/main.scss]: Noted. I removed both `!important`s in my last commit.

**Pull request closed without merging**